src/transfer/messages.rs

```diff
@@ -260,6 +260,7 @@ pub enum AutoAllocResponse {
     Events(Vec<AllocationEventHolder>),
     Info(Vec<Allocation>),
     List(AutoAllocListResponse),
+    ErrorResponse(String),
```

Comment:
Should it be HqError instead of a String?

Reply:
Please use `ToClientMessage::Error` instead of adding a new error response here.